src/strategies/spidStrategy.ts

```diff
@@ -129,6 +137,39 @@ const spidStrategy = (
     }
   };
 
+  const ARUBA_ID = "https://loginspid.aruba.it";
+  const INFOCERT_ID = "https://identity.infocert.it";
+  const INTESA_ID = "https://spid.intesa.it";
+  const LEPIDA_ID = "https://id.lepida.it/idp/shibboleth";
+  const NAMIRIAL_ID = "https://idp.namirialtsp.com/idp";
+  const POSTE_ID = "https://posteid.poste.it";
+  const SIELTE_ID = "https://identity.sieltecloud.it";
+  const SPIDITALIA_ID = "https://spid.register.it";
+  const TIM_ID = "https://login.id.tim.it/affwebservices/public/saml2sso";
+
+  const IDP_IDS: { [key: string]: string } = {
+    [ARUBA_ID]: "arubaid",
+    [INFOCERT_ID]: "infocertid",
+    [INTESA_ID]: "intesaid",
+    [LEPIDA_ID]: "lepidaid",
+    [NAMIRIAL_ID]: "namirialid",
+    [POSTE_ID]: "posteid",
+    [SIELTE_ID]: "sielteid",
+    [SPIDITALIA_ID]: "spiditalia",
+    [TIM_ID]: "timid"
+  };
+
+  const idpMetadata = await parseIdpMetadata(
```

Comment:
we also need to add the test IdP

Reply:
test idp is statically defined on [this](https://github.com/teamdigitale/io-backend/pull/420/files#diff-5c3aa1bec8f5930b168675af818efc7fR26) row